text/0000-interactive-app-creation.md

```diff
@@ -0,0 +1,142 @@
+
+In that context, this RFC seeks to help new Ember applications achieve [WCAG Success Criteria 3.1.1: language of page](https://www.w3.org/TR/WCAG21/#language-of-page) by providing users of `ember-cli` to specify the base human language of their new application within an interactive setup workflow.
+
+This RFC is the result of [analysis and discussion in Ember's accessibility strike team](https://github.com/ember-a11y/core-notes/blob/ember-a11y/ember-a11y/2020-05/may-20.md). The overall roadmap for addressing this issue is with a series of RFCs that intend to (independently) offer: 
+- a partial resolution to ensuring new Ember apps achieve [WCAG Success Criteria 3.1.1 (language of the page)](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html)
+- enhancements to Ember CLI that empower developers to make their applications more accessible-by-default
+- global improvements to new Ember applications (i.e., improvements gained by achieving WCAG SC-3.1.1 that aren't specifically related to digital accessibility)
+
+The first step of the solution was presented in [RFC 635](https://github.com/emberjs/rfcs/pull/635)-- the addition of the `--lang` flag as a way to add the language attribute to the `<html>` tag in the `index.html` file of new Ember apps. As such, successful implementation of [RFC 635](https://github.com/emberjs/rfcs/pull/635) (or similar) is a necessary pre-requisite for this RFC, which in turn proposes an interactive workflow that can be initiated in the CLI when creating a new Ember app. With this in mind, this RFC will endeavor to not repeat the information in that RFC, but rather build on it, and reference when appropriate. The intention of [RFC 635](https://github.com/emberjs/rfcs/pull/635) was to provide the appropriate underlying mechanism for setting the page language to align new Ember apps with WCAG Success Criteria 3.1.1 (language of the page); the interactive workflow proposed in **this** RFC intends to harness that mechanism.
+
+Lastly, this RFC is equally motivated by providing 'global' application improvements to the `ember new` workflow. Briefly, these improvements can be summarized as:
+- improvements to applications caused by achieving WCAG SC-3.1.1 that aren't strictly accessibility-related (as outlined in [RFC 635](https://github.com/emberjs/rfcs/pull/635))
+- flattening the learning curve for new users by providing them with an interactive environment and helpful feedback during troubleshooting, error handling, etc.
+
+## Detailed design
+
+### Prior Art
+It would be remiss to introduce this RFC without mentioning the prior art that we have for wizard-like app creation. In the Ember community, [@gossi](https://gist.github.com/gossi) first introduced this idea with [ember-cli-create](https://github.com/gossi/ember-cli-create). In the wider web community, the Vue community offers this through `vue create` ([see docs](https://cli.vuejs.org/guide/creating-a-project.html#vue-create)).
```

Comment:
```suggestion
It would be remiss to introduce this RFC without mentioning the prior art that we analyzed for wizard-like app creation. In the Ember community, [@gossi](https://gist.github.com/gossi) first introduced this idea with [ember-cli-create](https://github.com/gossi/ember-cli-create). In the wider web community, the Vue community offers this through `vue create` ([see docs](https://cli.vuejs.org/guide/creating-a-project.html#vue-create)).
```